modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java

```diff
@@ -2890,6 +2977,11 @@ private void _testBuildTemplateNpm(
 			"src/main/java/" + packagePath + "/constants/" + className +
 				"WebKeys.java");
 
+		if (Validator.isNotNull(System.getenv("JENKINS_HOME"))) {
```

Comment:
Can you please look for `JENKINS_HOME` in this class, and remove the `Assume.assumeFalse` lines? Otherwise, those tests won't even run on CI 😋